test/libsinsp_e2e/forking.cpp

```diff
@@ -336,7 +336,7 @@ static int clone_callback_1(void* arg)
 	// using a weird clone() here something goes wrong with
 	// recent version of glibc
 	ctid = syscall(SYS_getpid);
-	close(cp->fd);
+	printf("CHILD %d\n", close(cp->fd));
```

Comment:
leftover or should we display it?

Reply:
Sadly this does not fix the bug :/ I can't get a repro on my local setup and I'm trying to get some more info with the CI 😅 I'll remove this debug prints obv :)

Reply:
Thanks @therealbobo ! Could you please add `wip: ` to the PR title until you are done with experiments?

Reply:
sure!